### ISSUE TITLE
Bug with RHMC parameters passed as argument

### ISSUE THREAD

**Issue by clarkedavida** [LatticeQCD/SIMULATeQCD#50]
It seems that if you try to pass an RHMC parameter file as an argument, SIMULATeQCD will ignore it. This is inconvenient and not intended...

**Comment by clarkedavida** [comment]
There also apparently is an issue that parameter names are wrong. Maybe one can throw an error if an unrecognized parameter is wrong.

**Comment by clarkedavida** [comment]
The first issue can be solved by passing `argc` and `argv` as well. Will add to documentation, if it's not already there.

**Comment by lukas-mazur** [comment]
Related to this issue: These tests don't have a default parameter file:
```
main_DslashImagmuTest.cu
main_RndMultipleTest.cu
main_hisqSmearingMulti.cu
main_HBOR_single_test.cu
```
So a line such as 
```C++
 param.readfile(commBase, "../parameter/tests/<testname>.param", argc, argv);
```
is missing.

**Comment by luhuhis** [comment]
Was this partly resolved with #52 ?
